builders/integrations/wallets/rainbowkit.md

```diff
@@ -0,0 +1,328 @@
+
+    - This example uses [Next.js](https://nextjs.org/docs){target=\_blank} with Typescript and the App Router. You can find examples for additional frameworks in the [RainbowKit examples repository](https://github.com/rainbow-me/rainbowkit/tree/main/examples){target=\_blank}
+
+- A mobile wallet which supports Moonbeam out of the box or allows for adding custom networks
+
+    - This example uses [MetaMask](https://metamask.io/){target=\_blank}. However, RainbowKit supports a variety of wallets, and you can find a complete list on the RainbowKit [Custom Wallet List](https://www.rainbowkit.com/docs/custom-wallet-list){target=\_blank}
+
+- A WalletConnect `projectId` - Every dApp relying on WalletConnect is required to have an associated `projectId`. It is free to create an account, and you can instantly generate an ID. 
+
+To obtain a WalletConnect `projectId`:
+
+1. Visit [WalletConnect Cloud](https://cloud.walletconnect.com/){target=\_blank}
+2. On the **Projects** page, select **Create** 
+3. Add your project information (you can leave **Homepage URL** blank if you have not deployed your dApp)
+4. Select the **AppKit** SDK
+5. Select your coding environment or platform (select React for this guide)
+6. Locate your `projectId` in the left menu. You can also find it in the **Get started** code snippet of the WalletConnect Quickstart
+ 
+### Getting Started {: #getting-started }
+
+Ensure you are in the root directory for your project, then install RainbowKit and its peer dependencies:
+
+=== "npm"
+
+    ```bash
+    npm install @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm install @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+
+Next, start the development server to create a local dApp instance:
+
+=== "npm"
+
+    ```bash
+    npm run dev
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm run dev
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn dev
+    ```
+
+If you navigate to [http://localhost:3000](http://localhost:3000){target=\_blank},  you should see the starter Next.js application in your browser.
+
+To test the RainbowKit connection, you will use the MetaMask mobile app. To follow this guide, you must have established a connection to the Moonbase Alpha TestNet on the MetaMask mobile app.
+
+You can connect your MetaMask mobile wallet to the Moonbase Alpha TestNet in a couple of ways. You can manually add the Moonbase Alpha TestNet configurations from the **Networks** section of the **Settings** menu. Or you can also open up the **Browser** from MetaMask mobile and navigate to the [Moonbeam documentation site (docs.moonbeam.network)](/){target=\_blank}, click on **Connect MetaMask** at the top of the page, and select **Moonbase Alpha** from the menu. Follow the prompts to automatically add Moonbase Alpha as a custom network.
```

Comment:
Ooooh just realized the connect metamask button isn't available on mobile

Reply:
I honestly wish people would call it the MetaMask browser wallet because that's what we mean here.